packages/uswds-core/src/styles/mixins/general/table.scss

```diff
@@ -299,29 +289,31 @@ $table-unsorted-icon-color: get-color-token-from-bg(
 }
 
 @mixin usa-table--borderless {
```

Comment:
**question**: There's not a top border on the header. Is that intentional?

| Before | After |
| :----- | :---- |
| ![image](https://github.com/user-attachments/assets/2c928bf2-9c32-47bf-8a2e-507ce00d0bdf) [develop] |  ![image](https://github.com/user-attachments/assets/3e5776fa-5972-49b0-9880-25ae0086f6c3) [feature branch] |

[develop]: https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/?path=/story/components-table--borderless&p=all
[feature branch]: https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cb-table-headers-need-header-styling/?path=/story/components-table--default